src/libraries/Native/native-binplace.proj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
     <TargetFramework>$(BuildTargetFramework)</TargetFramework>
+    <TargetFramework Condition="'$(TargetFramework)' == ''">$(NetCoreAppCurrent)</TargetFramework>
```

Comment:
We have `<TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>`... that should work just fine if `BuildTargetFramework` is empty.